lib/Cargo.toml

```diff
@@ -13,7 +13,10 @@ keywords = ["leptos", "web", "framework", "reactive", "isomorphic"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-stable = []
+default = ["unboxed_closures", "fn_traits"]
+unboxed_closures = []
+fn_traits = []
+stable = ["leptos/stable"]
```

Comment:
This shouldn't be necessary. Changing the crate code to use `.set()` and `.get()` syntax as you did in `lib.rs` should be enough, as `leptos/stable` will be applied automatically if you enable the `stable` feature on `leptos`. As in, if you enable a feature for a crate, it's universally applied to all of your dependencies.

Reply:
Ah okay makes sense. I was building the crate from the repository root when I was resolving the error but when the crate is included as a dependency in another project it will build just fine without changes to the `Cargo.toml`. Thank you for the clarification!